From 724b07180c6bb5f862901bb7568e581757d2d296 Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 22 Jul 2020 13:56:08 +0700
Subject: [PATCH 268/433] ARM: dts: r8a77470: Add fdp0 node

Add Fine Display Processor (FDP) fdp0 node for the
Renesas FDP kernel module driver probe.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index 9ef99d8..213b847 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -1313,6 +1313,14 @@
 			};
 		};
 
+		fdp0: fdpm@fe940000 {
+			compatible = "renesas,fdp1";
+			reg = <0 0xfe940000 0 0x2400>;
+			interrupts = <0 262 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A77470_CLK_FDP1_0>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
+		};
+
 		vcp0: vcp0@fe900000 {
 			compatible = "renesas,vcp0";
 			reg = <0 0xfe900000 0 0x200>, <0 0xfe900200 0 0x200>;
-- 
2.7.4

